joomla/joomla-cms
# mod_feed - catch all Exceptions (Solves #9291)

PR to avoid displaying an error in mod_feed as issued in #9291.
The module should now render a message: _Feed not found_
### **Testing Instructions**
1. Add a Feed Display module
2. Add any RSS Feed which has uncommon date formats
3. See if the module throws error in Front-End.
